cohortextractor/cohortextractor.py

```diff
@@ -158,7 +160,17 @@ def _generate_cohort(
     index_date_range=None,
     skip_existing=False,
 ):
-    print("Running. Please wait...")
+    logger.info(
+        "Generating cohort for {study_name} in {output_dir}",
```

Comment:
Missing `f` string prefix here?

Side note: stdlib logging is [optimised](https://docs.python.org/3/howto/logging.html#optimization) when using `%` style formats.  I suspect we'll never see the benefit of it here but thought you could use just a smidge more logging info for the day!

Reply:
Well spotted!

Reply:
Using f strings is fine here IMO - it's clear, consistent, and negligible performance overhead is not a problem in this scenario.

FWIW, it's not %-style per se (you can use {} style if you've set a logging.Formatter with style='{') - it's the delayed string interpolation which is the optimisation.

e.g. If you do:

    logger.info('%s', 'msg')
    # or with style='{'
    logger.info('{}', 'msg')

Then the string will only be interpolated iff the log message is actually emitted by a handler. i.e. if it's a DEBUG level log, and you're only logging INFO, the interpolation will be skipped. Can be useful if you have a lot of DEBUG level logs with logs of embedded string interpolation - avoid the costs if not turned on.